src/pages/pages.scss

```diff
@@ -91,6 +96,25 @@ ul {
   line-height: 28px;
 }
 
+h2 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.breadCrumb {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+  color: $darkBlue;
+  display: flex;
+  margin-block-end: 0px;
+}
+
+.breadCrumb span {
+  align-self: center;
+}
+
 a {
   color: $brightBlue;
 }
```

Comment:
Would be better to call the .grayBackground class something else (i.e. pageHeading) and it should also have a padding of "16px 24px" while the title margin-block-end should be 0px.